src/BloomExe/Browser.cs

```diff
@@ -419,21 +419,23 @@ private void PasteFilteredText(bool removeSingleLineBreaks)
 
 		void OnShowContextMenu(object sender, GeckoContextMenuEventArgs e)
 		{
+			MenuItem FFMenuItem = null;
 			Debug.Assert(!InvokeRequired);
 			if (ContextMenuProvider != null)
 			{
-				ContextMenuProvider(e);
-#if DEBUG
-				e.ContextMenu.MenuItems.Add("Open Page in Firefox (which must be in the PATH environment variable)", new EventHandler(OnOpenPageInSystemBrowser));
-#endif
-				if (ContextMenuProvider(e))
-					return; // only the provider's items
+				var replacesStdMenu = ContextMenuProvider(e);
+				FFMenuItem = e.ContextMenu.MenuItems.Add("Open Page in Firefox (which must be in the PATH environment variable)",
```

Comment:
The PR exists because a code change caused a bunch of menu items to be in the menu twice.

The issue here is whether, in fixing it, we should make an additional change (as in the current code, perhaps inadvertently) to allow end users to see the "open page in firefox" command in secondary windows like the pages thumbnails, or whether that should continue to only be present in debug builds.

If it doesn't matter we can just merge as is.